localStorage_value_checker_sample_app/app.js

```diff
@@ -0,0 +1,43 @@
+(function() {
+  return {
+    events: {
+      // Framework Events
+      'app.created':'checkLocalStorage', // Call 'checkLocalStorage' function below
+      // Request Events (None)
+      // DOM events
+      'keyup #inputValueId': function(event){ // User clicked the 'enter'/'return' button so store value in localStorage
+        if(event.keyCode === 13)
+          return this.processInputValue();
+      },
+      'click .reset': 'resetValue' // Call 'resetValue' function below
+    },
+    checkLocalStorage: function () { // Look in localStorage for if value is present for 'exampleString'
+        if (this.store('exampleString') === null) { // If no value
+          services.notify('<span style="font-size: 14px;">localStorage value not set</span>', 'alert', 8000); // Services notification
+          this.switchTo('inputValue'); // Display input field for user to set value of localStorage
+        } else { // There is a value already set for 'exampleString' in localStorage
+          services.notify('<span style="font-size: 14px;">localStorage value found</span>', 'notice'); // Services notification
```

Comment:
When I move the styles into the app.css w a class on the inline span tags the styles aren't rendering - i.e. in this case the font size is not 14px - I've removed this entirely - the font size/spans.